fe/src/shared/components/Files/Files.js

```diff
@@ -1,31 +1,110 @@
-import React, { Fragment } from "react";
+import React, { Fragment, useState } from "react";
```

Comment:
**suggestion:** let's create a DownloadCenterFiles module for it to keep it maintainable.